tidb-cloud/import-csv-files.md

```diff
@@ -103,60 +103,46 @@ To import the CSV files to TiDB Cloud, take the following steps:
+    - **Import Schema and Data from S3** allows you to import SQL scripts for creating a table and import corresponding table data stored in S3 into TiDB.
 
-    - **Table Filter**: If you want to filter which tables to be imported, you can specify table filter rules in this area.
+6. If the source files do not meet the naming conventions, you can specify a custom mapping rule between a single target table and the CSV file. After that, the data source files will be re-scanned using the provided custom mapping rule. To modify the mapping, click **Advanced Settings** and then click **Mapping Settings**. Note that **Mapping Settings** is available only when you choose **Import into Pre-created Tables**.
 
-        For example:
+    You can also use wildcards to match the source files. For example:
 
-        - `db01.*`: all tables in the `db01` database will be imported.
-        - `!db02.*`: except the tables in the `db02` database, all other tables will be imported. `!` is used to exclude tables that do not need to be imported.
-        - `*.*` : all tables will be imported.
+    - `s3://<bucket-name>/path/my-data?.csv`: all CSV files starting with `my-data` and one character (such as `my-data1.csv` and `my-data2.csv`) in that folder will be imported into the same target table.
+    - `s3://<bucket-name>/path/my-data*.csv`: all CSV files in the folder starting with `my-data` will be imported into the same target table.
 
-        For more information, see [table filter syntax](/table-filter.md#syntax).
+    Note that only `?` and `*` are supported.
 
-5. Click **Next**.
+    - **Target Database**: enter the name of the target database you select.
 
-6. On the **Preview** page, you can have a preview of the data. If the previewed data is not what you expect, click the **Click here to edit csv configuration** link to update the CSV-specific configurations, including separator, delimiter, header, `backslash escape`, and `trim last separator`. For more information, see [CSV Configurations for Importing Data](/tidb-cloud/csv-config-for-import-data.md).
+    - **Target Tables**: enter the name of the target table you select. Note that this field only accepts one specific table name, so wildcards are not supported.
 
-    > **Note:**
-    >
-    > For the configurations of separator and delimiter, you can use both alphanumeric characters and certain special characters. The supported special characters include `\t`, `\b`, `\n`, `\r`, `\f`, and `\u0001`.
+    - **Source file URIs and names**: enter the source file URI and name, for example, `s3://sampledate/ingest/TableName.01.csv`. Note that the folder name of the URI must contain the data source folder. For more information about the URI format, see [URI format](https://docs.pingcap.com/tidb/stable/backup-and-restore-storages#uri-format).
```

Comment:
why linked to https://docs.pingcap.com/tidb/stable/backup-and-restore-storages#uri-format? I think it is inappropriate. Because we do not support `parameters` here.

Reply:
> Note that the folder name of the URI must contain the data source folder.

This note is submerged in the document does not seem to reflect its importance?

Reply:
Updated. PTAL.